code/modules/mob/living/simple_animal/hostile/wumborian_fugu.dm

```diff
@@ -129,21 +149,23 @@
 	item_flags = NOBLUDGEON
 	w_class = WEIGHT_CLASS_NORMAL
 	layer = MOB_LAYER
+	///A list of all the mob types unable to use this item.
 	var/list/banned_mobs
 
 /obj/item/fugu_gland/afterattack(atom/target, mob/user, proximity_flag)
 	. = ..()
-	if(proximity_flag && isanimal(target))
-		var/mob/living/simple_animal/A = target
-		if(A.buffed || (A.type in banned_mobs) || A.stat)
-			to_chat(user, "<span class='warning'>Something's interfering with [src]'s effects. It's no use.</span>")
-			return
-		A.buffed++
-		A.maxHealth *= 1.5
-		A.health = min(A.maxHealth,A.health*1.5)
-		A.melee_damage_lower = max((A.melee_damage_lower * 2), 10)
-		A.melee_damage_upper = max((A.melee_damage_upper * 2), 10)
-		A.transform *= 2
-		A.environment_smash |= ENVIRONMENT_SMASH_STRUCTURES | ENVIRONMENT_SMASH_RWALLS
-		to_chat(user, "<span class='info'>You increase the size of [A], giving it a surge of strength!</span>")
-		qdel(src)
+	if(!proximity_flag || !isanimal(target))
+		return
+	var/mob/living/simple_animal/animal_target = target
+	if(animal_target.buffed || (animal_target.type in banned_mobs) || animal_target.stat)
+		to_chat(user, "<span class='warning'>Something's interfering with [src]'s effects. It's no use.</span>")
+		return
+	animal_target.buffed++
+	animal_target.maxHealth *= 1.5
+	animal_target.health = min(animal_target.maxHealth, animal_target.health*1.5)
+	animal_target.melee_damage_lower = max((animal_target.melee_damage_lower * 2), 10)
+	animal_target.melee_damage_upper = max((animal_target.melee_damage_upper * 2), 10)
+	animal_target.transform *= 2
+	animal_target.environment_smash |= ENVIRONMENT_SMASH_STRUCTURES | ENVIRONMENT_SMASH_RWALLS
```

Comment:
btw, adding some extra object damage wouldn't be missed here, since it's kind of awkward for a fugu'd mob to be able to break down walls but not tables (due to having an obj_damage of 0).

Reply:
Not sure if that counts as a balance change